.github/workflows/build-m1-binaries.yml

```diff
@@ -113,9 +143,8 @@ jobs:
         shell: arch -arch arm64 bash {0}
         env:
           CONDA_PYTORCHBOT_TOKEN: ${{ secrets.CONDA_PYTORCHBOT_TOKEN }}
-          UPLOAD_CHANNEL: nightly
```

Comment:
Same as above, what will set a channel environment variable to `nightly` for regular builds?

Reply:
please see comment below we do set CHANNEL nightly by default